src/routes/docs/tutorials/apple/step-6/+page.markdoc

```diff
@@ -7,6 +7,66 @@ step: 6
 
 # Authentication {% #authentication %}
 
+In order to allow creating OAuth sessions, the following URL scheme must be added to your **Info.plist** file.
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+<dict>
+    <key>CFBundleTypeRole</key>
+    <string>Editor</string>
+    <key>CFBundleURLName</key>
+    <string>io.appwrite</string>
+    <key>CFBundleURLSchemes</key>
+    <array>
+        <string>appwrite-callback-[PROJECT_ID]</string>
+    </array>
+</dict>
+</array>
+```
+
+In `AppwriteService.swift`, let's update the object to include the auth methods by adding the following snippets.
+
+```swift
+    public func currentSession() async throws -> User<[String: AnyCodable]> {
```

Comment:
I think we should either change the name or the functionality of the function to match what is actually happening. The function is getting the current user as is, but if we want to show it getting the current session, we should use the snippet below:

```suggestion
func currentSession() async throws -> Session {
    try await account.getSession(
        sessionId: "current"
    )
}
```